nvidia/nvidia.go

```diff
@@ -0,0 +1,108 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2024 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package nvidia
+
+// ExceptionGlobs match libraries present in the driver package that are not expected to be used in snaps.
+//
+// In practice those are libraries loaded from the X11 server.
+var ExceptionGlobs = []string{
+	"nvidia/xorg/libglx.so*",
+	"nvidia/xorg/libglxserver_nvidia.so*",
+	"nvidia/xorg/nvidia_drv.so*",
+}
+
+// RegularGlobs match libraries present in the driver package that may be used in snaps.
```

Comment:
Should we mention that this should match content of `cmd/snap-confine/mount-support-nvidia.c` `nvidia_globs` which should not contain any of `ExceptionGlobs`? Should we perhaps test for this?

Reply:
This is now obsolete. The duplication is gone.